src/WebJobs.Script/Description/PowerShell/PowerShellFunctionInvoker.cs

```diff
@@ -87,7 +87,9 @@ public class PowerShellFunctionInvoker : ScriptFunctionInvokerBase
                         System.Management.Automation.PowerShell.Create())
                 {
                     powerShellInstance.Runspace = runspace;
-                    _moduleFiles = GetModuleFilePaths(_host.ScriptConfig.RootScriptPath, _functionName);
+                    _moduleFiles = FindDuplicateModules(GetModuleFilePaths(_host.ScriptConfig.RootScriptPath, _functionName), _host.ScriptConfig.RootScriptPath, _functionName);
+                    // Remove duplicates from _moduleFiles and only keep the ones closest to the Function in moduleDirectory
+
                     if (_moduleFiles.Any())
                     {
                         powerShellInstance.AddCommand("Import-Module").AddArgument(_moduleFiles);
```

Comment:
@mathewc this would be https://github.com/Azure/azure-webjobs-sdk-script/issues/897
Right now I just want to have a "super modules" directory that does not sit within one function. 
Let's say I have one module that I need to share across 5 functions in one Function App I only want it in one place, not in 5.
The next step will be to add implicit import as described in above issue which will address the issue you are having right now.